### PR TITLE
Update Readme to serve quickly the app

### PR DESCRIPTION
**Pull request by ger-benjamin**
To avoid reading doc each times.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -24,3 +24,16 @@ If you are interested in contributing or extending the project, take a look at t
 .. |Requirements Status| image:: https://requires.io/github/openoereb/pyramid_oereb/requirements.svg?branch=master
    :target: https://requires.io/github/openoereb/pyramid_oereb/requirements/?branch=master
    :alt: Requirements Status
+
+tl;nr - Local testing
```
Comment (jwkaltz) [inline README.rst:28]: Is this a typo? (The t1;nr part)

File: README.rst [hunk 2]
```diff
@@ -24,3 +24,16 @@ If you are interested in contributing or extending the project, take a look at t
...
+
+Run:
+
+.. code-block:: bash
+
+  git clone git@github.com:openoereb/pyramid_oereb.git
+  cd pyramid_oereb
+  make serve
+
+JSON embeddable extract is accessible at: http://localhost:6543/oereb/extract/embeddable/json/CH113928077734.
```
Comment (jwkaltz) [inline README.rst:39]: better to use the reduced flavour as "normal" testing flavour, in my opinion

**Review by jwkaltz — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (ger-benjamin) [reply README.rst]: => https://en.wiktionary.org/wiki/TL;NR

But perhaps that's not so common ? I have changed the title.

File: README.rst (hunk 2 above)
Comment (ger-benjamin) [reply README.rst]: Done

**Pull request merged**